integration-tests/mockApis/auth.js

```diff
@@ -191,10 +191,10 @@ const stubAuthUserRoles = () =>
     ],
   })
 
-const stubAuthAssignableRoles = () =>
+const stubAuthAssignableRoles = (body) =>
   getFor({
     urlPattern: '/auth/api/authuser/.*/assignable-roles',
-    body: [
+    body: body || [
```

Comment:
annoyingly `cy.task` seems to set `body` to `""` when not passed in so can't set a default on the input parameter.  Have to hack it here instead